### ISSUE TITLE
app crashing when starting video on iOS

### ISSUE THREAD

**Issue by jamesdixon** [Apparence-io/camera_awesome#56]
Hello!

## Steps to Reproduce

I copied your `_recordVideo` method pretty much verbatim.

```
  _recordVideo() async {
    if (_isRecordingVideo) {
      await _videoController.stopRecordingVideo();

      _isRecordingVideo = false;
      setState(() {});
    } else {
      final extDir = await getTemporaryDirectory();
      final testDir =
          await Directory('${extDir.path}/test').create(recursive: true);
      final filePath = '${testDir.path}/video_test.mp4';

      await _videoController.recordVideo(filePath);

      _isRecordingVideo = true;
      setState(() {});
    }
  }
```

## Expected results

Video should work.

## Actual results

App crashes and I see the following in the XCode console:

`*** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Unsupported value for standard codec'`

## About your device

| Brand   | Model       | OS        |
| ------- | ----------- | --------- |
| *Apple* | *iPhone 12 Pro*  | *14.2.1*  |

</details>

**Comment by g-apparence** [comment]
Thank you for reporting it.

**Comment by istornz** [comment]
Hi @jamesdixon, do you have a complete example to reproduce the error ?
The example provided in this repo works on your phone ?
Thanks !

**Comment by jamesdixon** [comment]
@istornz I will try and make a reproducible example!

**Comment by jamesdixon** [comment]
Hello! I was able to run the example successfully so not sure what's going on 🤔  It's a bit bizarre but I'll keep trying and let you guys know what I find. Thank you for all you do. Cheers

**Comment by jamesdixon** [comment]
Hello again! So, I copied the _exact_ example code directly into my own app and unfortunately, I get the same result. As soon as I tap the button to record video, the app crashes with the error mentioned above.

The good news is that I was able to reproduce the issue in your example. Basically, I copied in my dependencies and then ran your app and as soon as I started recording, the example app crashed. Here's a modified version of your `pubspec.yaml` that contains the dependencies from my app.

https://gist.github.com/jamesdixon/f335a8011a55a8e45d063c4eab600fdf

Let me know if you're able to reproduce with the same file.
